src/data/calculate.ts

```diff
@@ -231,14 +248,34 @@ export const calculatePersonRiskEach = (
   }
 }
 
+const getVaccineMultiplier = (data: CalculatorData): number | null => {
+  if (data.yourVaccineType === '') {
+    return 1
+  }
+  const vaccineMultipliers = Vaccines[data.yourVaccineType].multipliers
+  if (
+    data.yourVaccineDoses >= vaccineMultipliers.length ||
+    data.yourVaccineDoses < 0
+  ) {
+    return null
+  }
+  return vaccineMultipliers[data.yourVaccineDoses]
+}
+
 export const calculateActivityRisk = (data: CalculatorData): number | null => {
   try {
     if (data.interaction === '') {
       return null
     }
 
+    const vaccineMultiplier = getVaccineMultiplier(data)
+    if (vaccineMultiplier === null) {
+      console.log(data.yourVaccineType + ' ' + data.yourVaccineDoses)
```

Comment:
I assume you don't want this in anymore? Or do you want it removed once we finish the vaccine stuff? If so, we wanna add a noticeable comment next to it.

Reply:
Removed.